LICENSE

```diff
@@ -202,160 +202,221 @@
-For details see http://www.opensource.org/licenses/mit-license.php.
+This product bundles zTree, which is available under an MIT license.
+For details, see http://opensource.org/licenses/mit-license.php.
 
 ---------------------------------------------------------------------------
-jquery.tablednd (https://github.com/isocra/TableDnD)
+JSON (http://json.org)
 ---------------------------------------------------------------------------
-This product bundles jquery.tablednd which is available under an MIT license.
-For details see http://www.opensource.org/licenses/mit-license.php.
+This product bundles JSON, which is available under a BSD-like license.
+For details, see http://json.org/license.html.
 
 ---------------------------------------------------------------------------
-jquery.timeago (http://timeago.yarp.com/)
+Normalise.css (https://necolas.github.io/normalize.css/)
 ---------------------------------------------------------------------------
-This product bundles jquery.timeago which is available under an MIT license.
-For details see http://www.opensource.org/licenses/mit-license.php.
+This product bundles Normalise.css, which is available under an MIT license.
+For details, see http://opensource.org/licenses/mit-license.html.
 
 ---------------------------------------------------------------------------
-jquery.ztree (http://zTree.me/)
+Protocol Buffers (https://github.com/google/protobuf)
 ---------------------------------------------------------------------------
-This product bundles jquery.ztree which is available under an MIT license.
-For details see http://www.opensource.org/licenses/mit-license.php.
+This product bundles Protocol Buffers, which is available under a BSD
+license. For details, see http://opensource.org/licenses/BSD-3-Clause.
 
 ---------------------------------------------------------------------------
-jquery.multiselect (http://www.erichynds.com/jquery/jquery-ui-multiselect-widget/)
+Reset CSS (http://meyerweb.com/eric/tools/css/reset/)
 ---------------------------------------------------------------------------
-This product bundles jQuery.multiselect which is available under an MIT license.
-For details see http://www.opensource.org/licenses/mit-license.php.
+This product bundles Reset CSS, which is available in the public domain.
 
 ---------------------------------------------------------------------------
-jquery.ui (http://jqueryui.com/about)
+Respond (https://github.com/scottjehl/Respond)
 ---------------------------------------------------------------------------
-This product bundles jquery UI which is available under an MIT license.
-For details see http://www.opensource.org/licenses/mit-license.php.
+This product bundles Respond, which is available under an MIT license.
+For details, see http://opensource.org/licenses/mit-license.html.
+
+---------------------------------------------------------------------------
+tooltip.js (http://flowplayer.org/tools/tooltip/)
```

Comment:
Looks like a better location for this is https://github.com/jquerytools/jquerytools.